content/blog/webr-0-1-0/index.Rmd

```diff
@@ -0,0 +1,375 @@
+* [X] Find photo & update yaml metadata
+* [X] Create `thumbnail-sq.jpg`; height and width should be equal
+* [X] Create `thumbnail-wd.jpg`; width should be >5x height
+* [X] `hugodown::use_tidy_thumbnails()`
+* [x] Add intro sentence, e.g. the standard tagline for the package
+* [x] `usethis::use_tidy_thanks()`
+* [ ] Update temporary URLS to r-wasm.org
+* [ ] Change r-wasm/jupyterlite-webr-kernel GitHub repo public
+* [ ] Update georgestagg/webR repo for AWS and r-wasm.org URLs
+* [ ] Push npm update for r-wasm/webr package
+* [ ] Update post date
+-->
+```
+<!-- Initialise webR in the page -->
+
+`r hugodown::shortcode("webr-init")`
+
+<!-- Add webr engine for knit -->
+
+```{r, echo=FALSE}
+knitr::knit_engines$set(webr = function(options) {
+  code <- paste(options$code, collapse = "\n")
+  w <- knitr::opts_current$get('fig.width') * 72
+  h <- knitr::opts_current$get('fig.height') * 72
+  hugodown::shortcode("webr-editor", code = I(encodeString(code, quote = '`')), width = w, height = h)
+})
+```
+
+<!-- Blog post main content -->
+
+We're super excited to announce the release of webR v0.1.0! This is the first
+release of webR intended for general use by the web development and R
+communities and is the result of almost a year of hard work by the webR
+developers.
+
+This post will introduce webR, demonstrate some of the possibilities that
+running R in a web browser brings, and give a quick overview of how to include
+webR in your own TypeScript or JavaScript web applications.
+
+## Introduction
+
+WebR is a version of the open-source R interpreter compiled for WebAssembly,
+along with a supporting TypeScript library for interacting with the console and
+R objects from a JavaScript environment.
+
+By compiling R to WebAssembly a user can visit a website and run R code directly
+within the web browser, without R installed on their device or a supporting
+computational R server. All that is required is a normal web server, including
+the type of cloud hosting service provided by Github Pages or Netlify.
+
+## How it Works
+
+WebR's core is based around compiling the open-source R interpreter for
+[WebAssembly](https://webassembly.org), using the
+[Emscripten](https://emscripten.org) compiler suite along with [LLVM
+Flang](https://flang.llvm.org/docs/) to work with R's pre-existing C and Fortran
+based source code.
+
+WebAssembly (often abbreviated as Wasm) is a standard defining a virtual stack
+machine along with a corresponding *bytecode*. Efficient Wasm engines have
+already been implemented in most modern web browsers, which allows for the
+deployment of high performance Wasm applications on the web.
+
+While it's certainly possible for an interested programmer to write Wasm
+*bytecode* by hand, it is not required to do so. Similar to how code and data is
+compiled into *machine code* for a certain computer processor, code and data can
+be compiled into the Wasm *bytecode* by compiler software that supports the Wasm
```

Comment:
I found the repeated italics of bytecode and machine code slightly distracting. Maybe italicize just their first usages?